### DISCUSSION TITLE
Account Wrongfully Flagged Due to Mass Bot Reports - No Response for 45 Days

### DISCUSSION THREAD

**Discussion by cha-money** [community/community#130854] [Other Feature Feedback, Questions, & Ideas]
I am reaching out to seek urgent assistance regarding an issue with my GitHub account [github.com/snrat](https://github.com/snrat), which has been mistakenly flagged. I initially submitted a support ticket (Ticket #2807150) over 45 days ago but have yet to receive any response. Hence, I am now turning to the community for help as this prolonged silence is severely impacting my work.

LayerZero Labs has created a sybil-report project on their GitHub to collect reports from individual developers via Issues. Unfortunately, this initiative has led to unintended consequences. Many individuals, upon seeing their malicious activities being reported to LayerZero Labs through Issues, have started using mass reporting tools to target legitimate GitHub users who are submitting valid Issues.

I believe my account was flagged due to this misuse of mass reporting tools. I have never engaged in any malicious activities with my account. Moreover, there are already 5922 submissions blocked in the sybil-report project on [LayerZero Labs' GitHub](https://github.com/LayerZero-Labs/sybil-report/issues), suggesting that my account, along with many others, was likely flagged due to mass bot reports. You can review the 14 valid submissions I have made in this project, all of which comply with LayerZero Labs’ project guidelines.

Even if this use case of LayerZero Labs does not comply with GitHub's usage guidelines, I believe it is unfair to flag and suspend users who are submitting legitimate reports. Instead, GitHub should directly communicate with LayerZero Labs to resolve such issues.

My GitHub account has been registered for a very long time, and I have consistently used GitHub’s services, linking several important third-party accounts to it. In light of this situation and the fact that I have been waiting for over 45 days without any response, I earnestly request the GitHub team to manually review my account and assist in unblocking it as soon as possible.

GitHub states on their support ticket page that it may take more than 7 business days to receive a response. However, I submitted my appeal ticket on May 22, and I have not received any reply or assistance until today. My GitHub account is linked to many third-party services, which I can no longer use because my account has been flagged. Even worse, many accounts that use GitHub for login are now inaccessible to me.

I believe that, as a long-time GitHub user, many others also link numerous third-party services to their GitHub accounts. GitHub's lack of response is truly disappointing and leaves me feeling helpless. I hope the community can offer some advice on how to resolve this issue. Thank you, everyone.

**Comment by jackSeigerman** [comment]
did you get unflagged/ a response? I'm currently sitting at two weeks since making my ticket.

**Comment by wiizzoux** [comment]
Same problem. No response since May 7th (88 days!)

**Reply by cha-money** [reply]
I really don't know where all the customer service staff at GitHub have gone. Have they all been laid off?

**Comment by JoshuaDevCo** [comment]
been waiting almost 3 months... they are just a joke
